src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -30,7 +30,7 @@
     <Compile Include="OpenTests.cs" />
     <Compile Include="ImportExportTests.cs" />
     <Compile Include="PropertyTests.cs" />
-    <Compile Include="RSACngTests.cs" />
```

Comment:
Oh okay, gotcha. So these files were added sometime after the build was originally broken, I guess.